eclipse/src/saros/ui/wizards/CreateXMPPAccountWizard.java

```diff
@@ -86,66 +89,68 @@ public boolean performFinish() {
     cachedUsername = getUsername();
     cachedPassword = getPassword();
 
-    try {
-      // fork a new thread to prevent the GUI from hanging
-      getContainer()
-          .run(
-              true,
-              false,
-              new IRunnableWithProgress() {
-                @Override
-                public void run(IProgressMonitor monitor) throws InvocationTargetException {
-
-                  monitor.beginTask("Registering account...", IProgressMonitor.UNKNOWN);
-
-                  try {
-                    Registration registration =
-                        XMPPUtils.createAccount(cachedServer, cachedUsername, cachedPassword);
-
-                    if (registration != null)
-                      createAndThrowXMPPException(registration, cachedUsername);
-
-                    log.debug(
-                        "Account creation succeeded: username="
-                            + cachedUsername
-                            + ", server="
-                            + cachedServer);
-                  } catch (XMPPException e) {
-                    throw new InvocationTargetException(e);
-                  } finally {
-                    monitor.done();
-                  }
-                }
-              });
-    } catch (InvocationTargetException e) {
-      log.error(e.getCause().getMessage(), e.getCause());
-
-      String message = null;
-      Throwable t = e.getCause();
-
-      if (t instanceof XMPPException) message = getErrorMessage((XMPPException) t, cachedServer);
-
-      if (message == null && t != null) message = t.getMessage();
-
-      createXMPPAccountPage.setErrorMessage(message);
-
-      // Leave the wizard open
-      return false;
-    } catch (InterruptedException e) {
-      log.error("uninterruptible context was interrupted", e);
-      createXMPPAccountPage.setErrorMessage(e.getMessage());
-      return false;
-    }
-
-    // add account to the accountStore
-    this.createdXMPPAccount =
-        accountStore.createAccount(
-            cachedUsername, cachedPassword, cachedServer.toLowerCase(), "", 0, true, true);
-
-    if (createXMPPAccountPage.useNow())
-      SWTUtils.runSafeSWTAsync(
-          log, () -> XMPPConnectionSupport.getInstance().connect(createdXMPPAccount, true, false));
-
+    /* uncommented because create dialog is disabled and to avoid smack ref
```

Comment:
I know we already talked about why you commented out this code, but I am still not a big fan of it. If you don't want to have to deal with it during the smack upgrade, I would suggest just removing it. It will still be present in the version history, so we won't lose access to it either way.

Reply:
I argue that a grep on the code base will now not show this code and was not affecting readability.
But Ok for me, so I removed the code and made a comment linking a commit with the code.